README.md

```diff
@@ -199,8 +221,23 @@ monitoring dashboards, etc. see the [Deployments README](./deployments/README.md
 
 ### How to send a message
 
-A straightforward way to interact with and test the bridge is sending messages. This is explained
-in the [send message](./docs/send-message.md) document.
+```bash
+
+# send remark message from 
+./scripts/send-message-from-millau-rialto.sh remark
+```
+
+you will see logs as following showing transaction successfully sent to Millau node
+
+```
+...
+INFO bridge Sending message to Rialto. Size: 286. Dispatch weight: 1038000. Fee: 275,002,568
+INFO bridge Signed Millau Call: 0x7904710484d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0148b5009cfb8d63dc0aa61308f6ef85a570b94ad313425d5d6212e2309d9dd223b397742caed45e031e512b79fd225e3f298acced9841d381162583e9a8c6418e000c0000030000000001000000b0d60f000000000001d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d018eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a4801c85fb49849feb476c1c5acc5ea4a69d7532d4b6b1c082effbe56f80f9fad116927564e8b029932f4e6affbe7691fc287091ca0b218f1a3f5ad66051c838bcd830060070154556e69782074696d653a2031363234383036363535c834641000000000
+TRACE bridge Sent transaction to Millau node: 0x5e689fd157e6a30c3ed115700b7804fe7c29df4355d023996c4735374212bb010bfaacfed925b66f018ba2030d78c4f8dba2634c265a1646bb73bb51c2b4adf4
+```
+
+About how to interact with and test the bridge please check [send message](./docs/send-message.md) for detail explanation
+
```

Comment:
Similar to the other comment I made on this above, I would leave this at the beginning of the section so that readers have an introduction to what the section is going to cover.